block/manager.go

```diff
@@ -147,27 +147,37 @@ func (m *Manager) Start(ctx context.Context) error {
 		}
 	}
 
-	if !isSequencer {
-		// Fullnode loop can start before syncing from DA
-		go uevent.MustSubscribe(ctx, m.Pubsub, "applyGossipedBlocksLoop", p2p.EventQueryNewNewGossipedBlock, m.onNewGossipedBlock, m.logger)
-	}
-
 	// TODO: populate the accumulatedSize on startup
 
-	err = m.syncBlockManager()
-	if err != nil {
-		return fmt.Errorf("sync block manager: %w", err)
-	}
+	go m.refreshBlockSyncAdvertiseBlocks(ctx)
 
 	if isSequencer {
 		// Sequencer must wait till DA is synced to start submitting blobs
 		<-m.DAClient.Synced()
+		// Only sequencer must wait to be synced from DA.
```

Comment:
redundant comment since this in in the isSequencer branch

Reply:
removed